### PR TITLE
refactor: update votes page with new design

### PR DESCRIPTION
**Pull request by brenopolanski**
## Summary

Task: https://app.clickup.com/t/2h4e6e

## Checklist

- [ ] Documentation _(if necessary)_
- [x] Tests _(if necessary)_
- [x] Ready to be merged

### CODE REVIEW

File: src/app/components/EmptyBlock/EmptyBlock.tsx [hunk 1]
```diff
@@ -2,13 +2,16 @@ import React from "react";
...
 
-export const EmptyBlock = ({ className, message }: EmptyBlockProps) => (
+export const EmptyBlock = ({ className, children, message }: EmptyBlockProps) => (
 	<div
+		data-testid="EmptyBlock"
 		className={`border border-solid border-theme-neutral-300 dark:border-theme-neutral-800 p-6 rounded-lg text-theme-secondary-text ${className}`}
 	>
-		{message}
+		{children && <div>{children}</div>}
+		{message && <div>{message}</div>}
```
Comment (dated) [inline src/app/components/EmptyBlock/EmptyBlock.tsx:15]: There's no point in keeping `message` imho 🤔

File: src/domains/vote/components/AddressTable/AddressTable.tsx [hunk 2]
```diff
@@ -25,47 +31,75 @@ export const AddressTable = ({ wallets, onSelect }: AddressTableProps) => {
 		},
 		{
 			Header: t("COMMON.BALANCE"),
-			accessor: "balance",
-			className: "justify-end",
```
Comment (dated) [inline src/domains/vote/components/AddressTable/AddressTable.tsx:29]: Balances should always be aligned to the right afaik

File: src/app/components/EmptyBlock/EmptyBlock.tsx (hunk 1 above)
Comment (brenopolanski) [reply src/app/components/EmptyBlock/EmptyBlock.tsx]: done @dated :+1:

File: src/domains/vote/components/AddressTable/AddressTable.tsx (hunk 2 above)
Comment (brenopolanski) [reply src/domains/vote/components/AddressTable/AddressTable.tsx:29]: Fixed @dated

**Review by samharperpittam — APPROVED** [review]

**Comment by faustbrian** [comment]
@brenopolanski conflicts

**Review by faustbrian — APPROVED** [review]

**Pull request merged**